src/integrationTest/kotlin/net/corda/test/DriverBasedTest.kt

```diff
@@ -0,0 +1,26 @@
+package net.corda.test
+
+import net.corda.core.getOrThrow
+import net.corda.core.node.services.ServiceInfo
+import net.corda.node.services.transactions.SimpleNotaryService
+import net.corda.testing.DUMMY_BANK_A
+import net.corda.testing.DUMMY_BANK_B
+import net.corda.testing.DUMMY_NOTARY
+import net.corda.testing.driver.driver
+import org.junit.Test
+
+class DriverBasedTest {
+    private companion object {
+        val nodeALegalName = DUMMY_BANK_A.name
+        val nodeBLegalName = DUMMY_BANK_B.name
```

Comment:
Why are you splitting our the regular node names, but not the notary (see line 21)?

Reply:
Good point, a  holdover from the test I took the code from.